src/backend/crossterm.rs

```diff
@@ -59,9 +59,11 @@ impl Backend for CrosstermBackend {
         let cursor = crossterm::cursor();
         let mut last_y = 0;
         let mut last_x = 0;
+        let mut first = true;
         for (x, y, cell) in content {
-            if y != last_y || x != last_x + 1 {
+            if y != last_y || x != last_x + 1 || first {
```

Comment:
Rather than evaluate this condition for all iterations of the loop, wouldn't it be better to explicitly check for this case beforehand. Using something like `peek` on the iterator ?

```rust
        let mut content = content.peekable();
        if let Some((x, y, _)) = content.peek() {
            if *x == 1 && *y == 0 {
                string.push_str(&format!("{}", termion::cursor::Goto(x + 1, y + 1)));
                inst += 1;
            }
        }

```

Reply:
I didn't know about the internal branching of `Peekable::next`, thanks for pointing that out =). Then you make a totally valid point and I will merge as is.